hr_employee_firstname/models/hr_employee.py

```diff
@@ -0,0 +1,89 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright (C) 2010 - 2014 Savoir-faire Linux
+#    (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api, SUPERUSER_ID
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    def split_name(self, name):
+        new_name = [w for w in name.split(' ') if w]
```

Comment:
you can tell python how many parts you want: https://docs.python.org/2/library/stdtypes.html#str.split
so `name.split(' ', 1)` gives you at most two tokens